bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx128-na.html

```diff
@@ -0,0 +1,60 @@
+{#
+  This Source Code Form is subject to the terms of the Mozilla Public
+  License, v. 2.0. If a copy of the MPL was not distributed with this
+  file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% from "macros.html" import video_inline_embed with context %}
+{% from "macros-protocol.html" import picto with context %}
+
+{% extends "firefox/whatsnew/base.html" %}
+
+{% block page_title %}{{ ftl('whatsnew-page-title') }}{% endblock %}
+
+{#- This will appear as <meta property="og:description"> which can be used for social share -#}
+{% block page_og_desc %}{{ ftl('whatsnew-page-description') }}{% endblock %}
```

Comment:
OK, og sets `self.page_desc()`: https://github.com/mozilla/bedrock/blob/2656cbd0545f659970c496a36a6225d3cf1bad1a/bedrock/base/templates/base-protocol.html#L39

& meta leaves to the `page_desc` block:
https://github.com/mozilla/bedrock/blob/2656cbd0545f659970c496a36a6225d3cf1bad1a/bedrock/base/templates/base-protocol.html#L32

that is never set further (firefox/base or whatsnew/base) so that means empty description in this case.

Which leads me to adding `{% block page_desc %}{{ ftl('whatsnew-page-description') }}{% endblock %}` to the whatsnew/base should actually fix both the meta and og descriptions for all WNPs.

Reply:
This is a good suggestion. I think initially when these templates were started we had a different description for each page but somewhere along the way stopped doing that and it's been the same ever since. I'll do a follow-up PR to update the base template(s).